src/main/java/com/codeus/winter/annotation/QualifierAnnotationAutowireCandidateResolver.java

```diff
@@ -0,0 +1,56 @@
+package com.codeus.winter.annotation;
+
+import com.codeus.winter.config.DependencyDescriptor;
+import com.codeus.winter.util.AnnotationUtils;
+
+import java.lang.annotation.Annotation;
+import java.util.HashSet;
+import java.util.Set;
+
+public class QualifierAnnotationAutowireCandidateResolver {
```

Comment:
Perhaps, add a couple of simple tests for the `getSuggestedName` method.

Reply:
Added